libs/langchain-google-common/src/utils/gemini.ts

```diff
@@ -74,6 +83,29 @@ function messageContentImageUrl(
   }
 }
 
+function messageContentToMedia(
```

Comment:
Minor nit: Shouldn't this be `messageContentMedia` without the "to"? It is a "media" MessageContent type - we're not converting it to media.